CMakeLists.txt

```diff
@@ -87,6 +88,9 @@ include(version.cmake)
 include(cmake/git_version.cmake)
 
 
+configure_file(lib/CrashHandler/backtrace.h.in lib/CrashHandler/backtrace.h)
+include_directories(${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
I highly recommend to don't making binary root being included: Please make the sub-directory "generated-include" for such stuff (maybe something also could be done by this way)

```diff
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 211cebf..292a52e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -88,8 +88,9 @@ include(version.cmake)
 include(cmake/git_version.cmake)
 
 
-configure_file(lib/CrashHandler/backtrace.h.in lib/CrashHandler/backtrace.h)
-include_directories(${CMAKE_CURRENT_BINARY_DIR})
+configure_file(lib/CrashHandler/backtrace.h.in
+               generated-include/lib/CrashHandler/backtrace.h)
+include_directories(${CMAKE_CURRENT_BINARY_DIR}/generated-include)
 
 #if(WIN32 OR EMSCRIPTEN OR HAIKU)
 #    set(USE_SYSTEM_LIBS_DEFAULT OFF)
```

Reply:
I've just applied the suggested patch to the PR, good catch.

However, I'm not sure I understand what you mean by
> maybe something also could be done by this way

Are there any changes besides the suggested one I'm not seeing?

Reply:
I meant generally, when making any generated files, please put them at dedicated directory to guarantee no alien garbage (generated by build) will be messed up at the place.

All fine now, I merged 🦊👍